### ISSUE TITLE
Scrolling over the programselector doesn't work correctly

### ISSUE THREAD

**Issue by rtega** [asb2m10/dexed#448]
I'm using dexed on Linux. When I scroll on the programselector doesn't work the way it should. When I scroll down it jumps one program and then it jumps 2 programs each time I scroll. The same happens when I scroll up. This behavior is the same in the standalone program and in Reaper. I saw this in the previous version and in version 0.9.7.

It turns out these two lines are to blame in my case:
https://github.com/asb2m10/dexed/blob/482c8cd791d55667bdc4aa48f5914f6e659ae8aa/Source/DXComponents.cpp#L452
https://github.com/asb2m10/dexed/blob/482c8cd791d55667bdc4aa48f5914f6e659ae8aa/Source/DXComponents.cpp#L460

If set `accum_wheel = 0;` on these lines this is resolved.

Anybody else seeing this?

**Comment by asb2m10** [comment]
Yes, I can confirm this on Linux; I will try to find a more generic solution.
